### DISCUSSION TITLE
How about some better changelogs?

### DISCUSSION THREAD

**Discussion by mika76** [community/community#8822] [Copilot Conversations]
A bit more info could be useful - maybe examples...

**Comment by ollie-iterators** [comment]
I know, right? They have not updated the changelog since version 1.7, and now they are on version 1.11

**Reply by ollie-iterators** [reply]
And now it is even up to Version 1.12

**Reply by mika76** [reply]
Haha I think they're just trolling us at this point 😆

**Comment by DarthGigi** [comment]
Bump

**Comment by DarthGigi** [comment]
Any progress on this yet?

**Comment by davecheney** [comment]
Duplicate of https://github.com/orgs/community/discussions/109521

**Reply by mika76** [reply]
How can this issue, which I opened on Dec 14, 2021 be a duplicate of an discussion opened today?
